public/pages/CreateTrigger/containers/DefineTrigger/utils/validation.js

```diff
@@ -2,29 +2,19 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-import _ from 'lodash';
 
-import { FORMIK_INITIAL_TRIGGER_VALUES, TRIGGER_TYPE } from '../../CreateTrigger/utils/constants';
+export const validateTriggerName = (triggers = [], index, isFlyOut = false) => {
+  return (value) => {
+    const trimmedValue = value.trim();
+    if (!trimmedValue) return isFlyOut ? 'Required.' : 'Trigger name is required.';
```

Comment:
Nitpick: I believe`!trimmedValue` will return `true` if `value` is `0`. Could we refactor this check to [_.isEmpty](https://lodash.com/docs/4.17.15#isEmpty)?

Reply:
Per offline conversation, the `value` object is type `string`, not some number type, so it won't return `true` in this check. This is fine as-is; no need to revise.